tools/install.sh

```diff
@@ -163,6 +163,9 @@ mkdir -p ../project ../project/libraries ../project/instruments ../project/templ
 # Setting 777 permissions for templates_c
 chmod 777 ../smarty/templates_c
 
+# Changing group to www-data to give permission to create directories in Document Repository module
+sudo chown www-data.www-data ../modules/document_repository/user_uploads
```

Comment:
Yeah, I considered that. However it's quite a bit more complicated because we also still support Apache for Ubuntu installs...